src/typeScript/presentation/quizPage.ts

```diff
@@ -5,110 +5,151 @@ let quiz = questionShuffle(questions);
 let prozess = 1;
 let time = 0;
 let quizIsRunning = true;
-window.onload = function(){
+window.onload = function () {
     initilize();
     incQuestionIndex();
     //console.log(quiz);
 };
-   
-function initilize(){
+
+function initilize() {
     let element = document.getElementById("wrapQuestion");      //declares connection to a given div
```

Comment:
do fix the problem you can add a type assertion
like 
let element = document.getElementById("wrapQuestion") as HTMLInputElement;

Reply:
everything is an element but you can specify it with a name like renderQuestion or something else